lib/FieldType/Mapper/DateFormMapper.php

```diff
@@ -36,4 +47,40 @@ public function mapFieldDefinitionForm(FormInterface $fieldDefinitionForm, Field
                 ]
             );
     }
+
+    /**
+     * @param FormInterface $fieldForm
+     * @param FieldData $data
+     */
+    public function mapFieldValueForm(FormInterface $fieldForm, FieldData $data)
+    {
+        $fieldDefinition = $data->fieldDefinition;
+        $formConfig = $fieldForm->getConfig();
+        $names = $fieldDefinition->getNames();
+        $label = $fieldDefinition->getName($formConfig->getOption('languageCode')) ?: reset($names);
```

Comment:
it's the same for all mappers, which makes me think this should be extracted and reused instead of duplicated in every mapper